config.json.sample

```diff
@@ -0,0 +1,5 @@
+{
+  "debug": false,
+  "featureManagerIsFullZoneProvisioningEnabled": true,
+  "locale": "en"
+}
```

Comment:
Should we add that in the README. Instead of having this file that is never used?